src/api/config/routes.rb

```diff
@@ -509,6 +509,7 @@ def self.public_or_about_path?(request)
 
     ### /trigger
     post 'trigger/runservice' => 'trigger#runservice'
+    post 'trigger/service/github' => 'services/github_webhooks#create'
```

Comment:
The route should contain something about the kind of action, like "runservice". 
Other actions, like branching on pull request should also be possible.

Reply:
k, the "kind of action" is part of the hook header, so one route should be enough. We should maybe name the route 'trigger/webhook/github' to avoid confusion with source services?

And the code should filter for the even type and trigger services only on commits